opendevin/sandbox/docker.py

```diff
@@ -6,31 +6,31 @@
 import shlex
 import select
 import subprocess
-from typing import List
+from typing import List, Optional
```

Comment:
```suggestion
from typing import List
```

We shouldn't need `Optional` since we can just use `| None` instead.

(we also don't need `List` because just `list` works, but we can fix that later)